### DISCUSSION TITLE
doris-3.0.1   导入数据报错

### DISCUSSION THREAD

**Discussion by manbuyunduan666** [apache/doris#40524] [A - General / Q&A]
当我尝试在集群中执行sql 语句时，集群会挂掉，按照顺序挂掉。
![49d3791f93bae55cf1bf22a9e560e8b](https://github.com/user-attachments/assets/f993d094-5412-4013-97e5-5b5d02a73bff)

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。
